MainRepository/Program.cs

```diff
@@ -1,28 +1,34 @@
-﻿// Implementation of string is valid file path or not into tuple format.
+﻿// This program validates the file path
+// BNF diagram of finding valid file path
+// file://C:/WorkGIT/Assignments/MainRepository/Data/bnf-diagram.PNG
 using static State;
 var strs = new List<string> { @"C:\PROGRAMS\DATA\README.TXT", @"C:\PROGRAMS\README.TXT", @"C:\PROGRAMS\DATA\DIR\README.TXT", @"C:\README.TXT", "..", "", "C:", "C:\\PROGRAMS\\README..TXT" };
 foreach (var str in strs) {
-   if (ParseFile (str)) {
-      Console.WriteLine ($"{str} is a Valid filepath.");
-      var res = Result (str);
-      Console.WriteLine ($"{res.dir}, {res.folder}, {res.filename}, {res.ext}");
-   } else { Console.WriteLine ($"{str} is Not a Valid filepath."); }
+   if (IsValidFilePath (str)) {
+      Console.ForegroundColor = ConsoleColor.Green;

```

Comment:
These 3 lines can be moved to a method and called here, as they are repeated in if and else parts